gusto/equations/shallow_water_equations.py

```diff
@@ -13,24 +13,26 @@
 )
 from gusto.equations.prognostic_equations import PrognosticEquationSet
 
+
 __all__ = ["ShallowWaterEquations", "LinearShallowWaterEquations",
+           "ThermalShallowWaterEquations",
+           "LinearThermalShallowWaterEquations",
            "ShallowWaterEquations_1d", "LinearShallowWaterEquations_1d"]
 
 
 class ShallowWaterEquations(PrognosticEquationSet):
     u"""
     Class for the (rotating) shallow-water equations, which evolve the velocity
     'u' and the depth field 'D', via some variant of:                         \n
-    ∂u/∂t + (u.∇)u + f×u + g*∇(D+b) = 0,                                      \n
+    ∂u/∂t + (u.∇)u + f×u + g*∇(D+B) = 0,                                      \n
     ∂D/∂t + ∇.(D*u) = 0,                                                      \n
-    for Coriolis parameter 'f' and bottom surface 'b'.
+    for Coriolis parameter 'f' and bottom surface 'B'.
     """
 
     def __init__(self, domain, parameters, fexpr=None, bexpr=None,
```

Comment:
Should we change `bexpr` to `Bexpr` or something like `topog_expr` to help avoid confusion?

Reply:
yes